winlogbeat/Makefile

```diff
@@ -12,7 +12,3 @@ gen:
 # This is called by the beats packer before building starts
 .PHONY: before-build
 before-build:
-	# Windows
-	sed -i 's|#\{0,1\}\(to_files:\).*|\1 true|' $(PREFIX)/$(BEATNAME)-win.yml
-	sed -i 's|#\{0,1\}\(level:\).*|\1 info|' $(PREFIX)/$(BEATNAME)-win.yml
-	sed -i '/log files/{n;s|#\{0,1\}path:.*|path: C:/ProgramData/winlogbeat/Logs|}' $(PREFIX)/$(BEATNAME)-win.yml
```

Comment:
The only thing I will miss here is having the info level logging out of the box. When you are new first time user it's nice to have some positive feedback in the logs that its running and indexing events.